public/app/features/folders/FolderSettingsPage.tsx

```diff
@@ -66,10 +67,13 @@ export class FolderSettingsPage extends PureComponent<Props, State> {
     evt.stopPropagation();
     evt.preventDefault();
 
+    const confirmationText = `Do you want to delete this folder and all its dashboards${
+      config.featureToggles.ngalert ? ' and its Grafana 8 Alerts' : ''
```

Comment:
```suggestion
      config.featureToggles.ngalert ? ' and alerts' : ''
```
Small nit, I'd make this simpler. Wether dashboard alerts or grafana 8 alerts, they're getting deleted either way :)

Reply:
It makes sense. I will remove also the conditional: `config.featureToggles.ngalert ?` in that case.